CollapseLauncher/XAMLs/MainApp/Pages/SettingsPage.xaml.cs

```diff
@@ -72,12 +73,16 @@ public sealed partial class SettingsPage : Page
         private readonly string ExplorerPath =
             Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "explorer.exe");
 
+        private DnsSettingsContext DnsSettingsContext;
```

Comment:
Add `// ReSharper disable once NotAccessedField.Local` to ignore this error
Also rename the var as `_dnsSettingsContext` as its a private var